sonic-xcvrd/xcvrd/dom/utilities/vdm/utils.py

```diff
@@ -38,7 +38,6 @@ def get_vdm_thresholds(self, physical_port):
         try:
```

Comment:
@mihirpat1 why this change for only threshold and not for others?

Reply:
@prgeor For other APIs, the caller is invoking `is_transceiver_vdm_supported` to ensure VDM is supported.
However, xcvrd currently does not check for VDM support before calling `get_vdm_thresholds`